cmd/root.go

```diff
@@ -28,6 +28,7 @@ var (
 	flagConfiggin      string
 	flagLightOpinions  string
 	flagDarkOpinions   string
+	flagJSON           bool
```

Comment:
Sure. Note however that currently *only* `show properties` reacts to this flag in any way, shape, or form.